cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -1467,6 +1478,21 @@ pFilterByStakeAddress =
       <> Opt.help "Filter by Cardano stake address (Bech32-encoded)."
       )
 
+pByronAddress :: Parser (Address Byron)
+pByronAddress =
+    Opt.option
+      (Opt.eitherReader deserialise)
+        (  Opt.long "address"
+        <> Opt.metavar "STRING"
+        <> Opt.help "Byron address (Base58-encoded)."
```

Comment:
Improve this help text to explain why one might want to specify a Byron address when constructing a bootstrap witness.